dataland-frontend/src/utils/DataFormatUtils.ts

```diff
@@ -49,6 +49,7 @@ export function calculateExpiryDateAsDateString(expiryTimeDays: number): string
  * @returns the hyphenated date string
  */
 export function getHyphenatedDate(date: Date): string {
-  const delocalizedTime = date.getTime() - date.getTimezoneOffset() * 60 * 1000;
+  const newDate = new Date(date.toString());
+  const delocalizedTime = newDate.getTime() - newDate.getTimezoneOffset() * 60 * 1000;
   return new Date(delocalizedTime).toISOString().substring(0, 10);
 }
```

Comment:
Some Questions:

1.) Why do we need this in general? How come there is a timetone offset?
2.) Why do you convert `date`, which already has the type `Date`, to a string, and then again to a `Date` in line 52?

Reply:
added by Jan for some kind of typeScript issue, though now it doesn't seem to exist. I went back to the previous version

Reply:
Ok that resolves point 2.

But I'm still confused why the method is called "getHyphenatedDate" and the descriptions also just tells that it hypehnates a date to the fromat (yyyy-MM-dd), while in fact it also does subtract a timeZoneOffset.

Should I ask Jan why we need this?

Reply:
According to git "date.getTimezoneOffset()" was added by Marc during the task: "Implemented Version History Tracking for datasets uploaded to Dataland!" so i didn't change it.

Reply:
Discussed with Florian and refactored.
Substracting the timezone offset is in fact needed because of the `toISOString()` method which takes the timezone offset into account. So to have a delocalized calculation of the hyphenated date, we need to do this operation.